hugo/static/css/style.css

```diff
@@ -276,4 +284,45 @@ footer a {
 
 .fg-black {
     color: #000000;
+}
+
+.button {
+    padding: 0.5rem 0.75rem;
+    font-family: "Barlow", sans-serif;
+    color: #FFFFFF;
+    font-size: 1.2rem;
+    font-weight: 500;
+    background-color: #643B88;
+    cursor: pointer;
+    border-radius: 0.75rem;
+    display: flex;
+    align-items: center;
+    border-style: none;
+}
+
+.button img {
+    width: 1.75rem;
+    height: 1.75rem;
+    padding-right: 0.5rem;
+}
+
+.login-form {
+    font-family: "Urbanist", sans-serif;
+    display: flex;
+    gap: 1.5rem;
+    align-items: center;
+    justify-content: center;
+}
+
+@media (max-width:768px) {
+    .login-form {
+        flex-direction: column;
+    }
+}
+
+.input-field {
```

Comment:
just personal preference, but would give the input fields a little bit of padding. Also i would always allign them in a column. Not just when on mobile.